README.md

```diff
@@ -122,7 +123,67 @@ If you want to deploy directly from the CLI, run `yarn vercel` and follow the st
 
 If you want to redeploy to the same production URL you can run `yarn vercel --prod`. If you omit the `--prod` flag it will deploy it to a preview/test URL.
 
-**Make sure your `packages/nextjs/scaffold.config.ts` file has the values you need.**
+**Make sure to check the values of your Scaffold Configuration before deploying your NextJS App.**
+
+### Scaffold App Configuration
+
+You can configure different settings for your dapp at `packages/nextjs/scaffold.config.ts`.
+
+```ts
+export type ScaffoldConfig = {
+  targetNetwork: chains.Chain;
+  pollingInterval: number;
+  alchemyApiKey: string;
+  walletConnectProjectId: string;
+  onlyLocalBurnerWallet: boolean;
+  walletAutoConnect: boolean;
+  // your dapp custom config, eg:
+  // tokenIcon : string;
+  // eventName : string;
+  // hideHeader : boolean;
+};
+```
+
+The configuration parameters are described below, make sure to update the values according to your needs:
+
+- **targetNetwork**  
+  Sets the blockchain network where your dapp is deployed. Use values from `wagmi/chains`.
+
+- **pollingInterval**  
+  The interval in milliseconds at which your front-end application polls the RPC servers for fresh data. _Note that this setting does not affect the local network._
+
+- **alchemyApiKey**  
+  Default Alchemy API key from Scaffold ETH 2 for local testing purposes.  
+  It's recommended to obtain your own API key from the [Alchemy Dashboard](https://dashboard.alchemyapi.io/) and store it in an environment variable: `NEXT_PUBLIC_ALCHEMY_API_KEY` at `\packages\nextjs\.env` file.
+
+- **walletConnectProjectId**  
+  WalletConnect's default project ID from Scaffold ETH 2 for local testing purposes.
+  It's recommended to obtain your own project ID from the [WalletConnect website](https://cloud.walletconnect.com) and store it in an environment variable: `NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID` at `\packages\nextjs\.env` file.
+
+- **onlyLocalBurnerWallet**  
+  Controls the networks where the Burner Wallet feature is available. This feature provides a lightweight wallet for users.
+
+  - `true` => Use Burner Wallet only on hardhat network.
+  - `false` => Use Burner Wallet on all networks.
+
+- **walletAutoConnect**  
+  Set it to true to activate automatic wallet connection behavior:
+  - If the user was connected into a wallet before, on page reload it reconnects automatically.
+  - If user is not connected to any wallet, on reload, it connects to the burner wallet if `burnerWallet.enabled` is true and `burnerWallet.onlyLocal` is false.
```

Comment:
These values were removed. It should be something like

If user is not connected to any wallet, on reload, it connects to the burner wallet if it is enabled for the current network. See `onlyLocalBurnerWallet`